packages/core/data-transfer/lib/engine/index.ts

```diff
@@ -1,9 +1,12 @@
+import { isEmpty, uniq } from 'lodash/fp';
```

Comment:
Let's try to keep a convention for the imports.

My suggestion would be 

```typescript
import external types
import project types

import external libraries

import project files
```

What do you think?

Reply:
I agree, but it's difficult to remember and implement manually. Any idea if linting rules can take care of that, or do they only exist for module vs file imports?

Reply:
Oh, this could partly be because we're not yet linting .ts. Once https://github.com/strapi/strapi/pull/14716 is merged, most of this issue might be fixed.